### PR TITLE
test: add edge case tests for flag parsing

### LINKED ISSUE #41: test: add edge case tests for flag parsing
## Description

Several edge cases in command flag parsing are untested:

1. **Comma-separated values with whitespace** — e.g., `--package " git-workflow , threat-modelling "`
2. **Empty items after comma split** — e.g., `--package "git-workflow,,foo"`
3. **Combined flags** — `--package` + `--standards` + `--for` used together in a single install/uninstall/update command
4. **Invalid `--for` flag** — e.g., `--for vscode` (not a valid assistant)
5. **`update` with nothing installed** — currently returns success (`nil`), should this be an error?

## Proposed Changes

Add test cases covering these edge cases to the respective `_test.go` files in `internal/cmd/`.

## Acceptance Criteria

- All edge cases above have at least one test
- Behavior for each edge case is explicitly documented (error vs. success)

### PR DESCRIPTION
**Pull request by willvelida**
Add regression tests for flag parsing edge cases that are handled correctly but previously untested.

## Changes

- **buildPackageList** (2 table rows): whitespace around package names is trimmed, empty items from double commas are skipped
- **Uninstall**: invalid --for value returns error mentioning the invalid name and listing valid assistants
- **Update**: same invalid --for pattern

## Coverage

internal/cmd package: 86.8% -> 87.8%

## Not in scope

- --standards flag (removed from codebase)
- update with nothing installed (already tested)

Closes #41

### CODE REVIEW

File: internal/cmd/update_test.go [hunk 1]
```diff
@@ -130,6 +130,33 @@ func TestUpdateInvalidPackageReturnsError(t *testing.T) {
...
+	if err == nil {
+		t.Fatal("expected error for unknown assistant, got nil")
+	}
+
+	msg := err.Error()
+	if !strings.Contains(msg, "vscode") {
+		t.Errorf("error should mention the invalid name: got %q", msg)
+	}
+	if !strings.Contains(msg, "copilot") || !strings.Contains(msg, "claude") {
+		t.Errorf("error should list valid assistants: got %q", msg)
```
[the remark this answers is not in the document]
Comment (willvelida) [reply internal/cmd/update_test.go]: Good catch — replaced with a loop over assistant.List() in 1abc7da. Tests now stay correct when assistants are added or removed.

File: internal/cmd/uninstall_test.go [hunk 2]
```diff
@@ -120,6 +121,31 @@ func TestUninstallForClaude(t *testing.T) {
...
+	if err == nil {
+		t.Fatal("expected error for unknown assistant, got nil")
+	}
+
+	msg := err.Error()
+	if !strings.Contains(msg, "vscode") {
+		t.Errorf("error should mention the invalid name: got %q", msg)
+	}
+	if !strings.Contains(msg, "copilot") || !strings.Contains(msg, "claude") {
+		t.Errorf("error should list valid assistants: got %q", msg)
```
[the remark this answers is not in the document]
Comment (willvelida) [reply internal/cmd/uninstall_test.go]: Same fix applied here — 1abc7da.
